code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -9,6 +9,7 @@
 	ammo_type = list(/obj/item/ammo_casing/energy/lasergun)
 	ammo_x_offset = 1
 	shaded_charge = TRUE
+	can_holster = TRUE
```

Comment:
Changing it this high up might have adverse consequences. 

That and I don't think that fullsize laser guns should be holsterable. A holster (to me anyway) is designed for pistol-sized (class? I'm not in the US so correct me on terminology), and given we have a lot fewer of those than larger guns, I would rather we override on just those instead of this high up.